src/QMCWaveFunctions/Fermion/MultiSlaterDetTableMethod.h

```diff
@@ -186,6 +187,12 @@ class MultiSlaterDetTableMethod : public WaveFunctionComponent, public Optimizab
 
   void evaluateDerivativesWF(ParticleSet& P, const opt_variables_type& optvars, Vector<ValueType>& dlogpsi) override;
 
+  void evaluateDerivativesWF_local(Vector<ValueType>& dlogpsi) override;
+
+  void detRatios(Vector<ValueType>& ratios) override;
```

Comment:
Please document what this function do. Also longer and more descriptive name is preferred. This function should probably be a const member.

Reply:
I will note it is already the length of similar functions nearby.  The function above it is essentially `derivativesWF` since `evaluate` is superfluous.

Reply:
However, state your preference on length and I will create a corresponding name.

Reply:
I saw you were calculating `d log(Psi)/ d c_i`
I would recommend `calcLinearExpansionDeriv`, for the following reasons:
1. functions should start with a verb. See developer guidelines.
2. In MSD, each linear expansion term is a multiplication of one or more determinants, most of the time two. So never call things determinants which are not determinants.